.github/workflows/dart.yml

```diff
@@ -61,15 +61,17 @@ jobs:
       matrix:
         os: [macos-13, macos-14, macos-15]
         package: [sign_in_with_apple/sign_in_with_apple]
-        xcode: ["14.3.1", "15.2", "16.0"]
+        xcode: ["14.3.1", "15.2", "16.1"]
```

Comment:
Personally I would like to use at least 16.1 and 16.2 (where the new case was added and what you check with `compiler(>=6.0.3)`, right?).

I don't think we can expect users (and their CI) to always use the very latest Xcode version shortly after release, so we have to be sure to support the currently expected ones.

Reply:
These changes were not part of my original PR, but changes already merged in https://github.com/aboutyou/dart_packages/pull/456.

I have removed these changes from my PR, so now it should be correct. 

As for the check, i think it would be good to also check for 16.2, so if wanted, i can add that to my PR.

Reply:
> As for the check, i think it would be good to also check for 16.2, so if wanted, i can add that to my PR.

Yes, that's what I would also suggest, given that the change you added (with that specific compiler version) will only be included in 16.2.

Reply:
I have added 16.2 for both macos and ios build.